dogfood/Dockerfile

```diff
@@ -312,8 +312,19 @@ COPY --from=go /tmp/bin /usr/local/bin
 COPY --from=rust-utils /tmp/bin /usr/local/bin
 COPY --from=proto /tmp/bin /usr/local/bin
 
+# Configure Nix without sandboxing - https://github.com/NixOS/nix/issues/2636#issuecomment-455302745
+RUN mkdir -p /nix /etc/nix \
+ && chmod a+rwx /nix \
+ && echo 'sandbox = false' > /etc/nix/nix.conf
+
 USER coder
 
+# Install Nix
+RUN sh <(curl -L https://nixos.org/nix/install) --no-daemon
```

Comment:
Is there a less `curl | bash` way of installing Nix? You can check in other files like keys or apt repos if you need if that helps

Reply:
Unfortunately there isn't.

Reply:
Maybe you can pin a specific version instead of the latest? For example, you could download it from: [link](https://releases.nixos.org/?prefix=nix/nix-2.9.2/)

Reply:
If it isn't a big effort, I would go for it.

Reply:
✅
🎶 https://www.youtube.com/watch?v=oRdxUFDoQe0